zernikedlg.h

```diff
@@ -27,7 +27,7 @@
 #include <vector>
 #include <QVector>
 
-#define Z_TERMS 48
+#define Z_TERMS 49
```

Comment:
```suggestion
#define Z_ORDER 7
#define Z_TERMS (Z_ORDER*Z_ORDER) 
```
I suggest this for code readability and prevent futur errors. Feel free to adjust. 
Comment is also welcome. For example to remember that all terms of a given order should be used.

Reply:
Oops.  just merged.

It's more complicated.  Order is 12 (not 7).  It would be like this:

#define Z_TERMS (Z_ORDER/2+1)*(Z_ORDER/2+1)


Note that orders are always even numbers.  Z_TERMS are always perfect squares

Maybe just add a comment with the math right there at the Z_TERMS code?

Reply:
There are several places where this math are done. Ideally there would be some helper function and necessary defines somewhere.  
Let's just add a comment here to start with